how-tos/maintain/upgrade-notes-rails/upgrade-to-v6.md

```diff
@@ -13,6 +13,10 @@ Please be aware that while Forest Admin make every effort to ensure that our pla
 
 {% endhint %}
 
+{% hint style="info" %}
+Upgraded your agent but changes are not reflecting on your UI ? [Follow this note](../push-your-new-version-to-production.md).
```

Comment:
I think we should always push to this docs, even if there are no visible problem in the UI.

So I would prefer: "Please follow the recommended procedure regarding forestadmin-schema". Or something like that.